<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>324</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);
}


QPushButton:hover{
       border-color: rgb(50, 50, 50);
border-style: solid;
border-width: 1px;
border-radius: 3px;
background-color: rgb(50, 50, 50);
}

QPushButton {
       border-color: rgb(50, 50, 50);
border-style: solid;
border-width: 1px;
border-radius: 3px;
}




QPushButton:pressed {
       border-color: rgb(100, 100, 100);
border-style: solid;
border-width: 1px;
border-radius: 3px;
	background-color: rgb(70, 70, 70);

}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="L_U">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>U</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_U">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(50, 50, 50);
border-style: solid;
border-width: 1px;
border-radius: 3px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_UDef">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_UClear">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_USort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="L_Res">
        <property name="minimumSize">
         <size>
          <width>510</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(50, 50, 50);
border-style: solid;
border-width: 1px;
border-radius: 3px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Calculate">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="L_B">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_B">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(50, 50, 50);
border-style: solid;
border-width: 1px;
border-radius: 3px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CB_BInvert">
        <property name="text">
         <string>Invert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_BClear">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_BSort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="COMB_Oper">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(50, 50, 50);
border-style: solid;
border-width: 1px;
border-radius: 3px;</string>
      </property>
      <item>
       <property name="text">
        <string>⋃</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>⋂</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>\</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>△</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="L_A">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_A">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(50, 50, 50);
border-style: solid;
border-width: 1px;
border-radius: 3px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CB_AInvert">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Invert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_AClear">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_ASort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
